README.md

```diff
@@ -1 +1,186 @@
-# capstone-project1
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <h3><b>Capstone project 1</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Capstone project 1] <a name="about-project"></a>
+
+
+**[Capstone project 1]** Is a personal portfolio made made from scratch using Html, Css and Flexbox.
+
+## 🛠 Built With <a name="built-with"></a>
+
+HTML5, CSS3, JavaScript, Git and Github
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://lenguajehtml.com/html/">HTML5</a></li>
+    <li><a href="https://lenguajecss.com/">CSS3</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Best organization]**
+- **[Professional documentation]**
+- **[Clean Working]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://bohaz.github.io/capstone-project1/)
```

Comment:
- [ ] Great job deploying your live version of your project as suggested however the live version is not working. 
- I think the reason for this is because you deployed the `main` instead of `capstone-branch`. Please kindly check this.

<img width="960" alt="image" src="https://github.com/bohaz/capstone-project1/assets/52934128/d07ccd08-4399-4575-9f92-e8a6b845170d">


- [ ] Also, Please kindly ensure that after properly deploying the project, all pages, links, and images work as expected.